test-load/pkg/cmd/dolphin/tests/load.go

```diff
@@ -1,74 +1,85 @@
-/*
```

Comment:
I understand this already merged, but hope we can start the discussion here then address it in the following PR.

Why the copyright been removed here? also most of other source files in the PR don't have copyright file.